src/main/kotlin/pl/umcs/picto3/game/InMemoryGame.kt

```diff
@@ -0,0 +1,14 @@
+package pl.umcs.picto3.game
+
+import pl.umcs.picto3.gameconfig.GameConfig
+import pl.umcs.picto3.player.Player
+import pl.umcs.picto3.round.InMemoryRound
+import java.util.UUID
+import java.util.concurrent.ConcurrentHashMap
+
+
+data class InMemoryGame(
+    var gameConfig: GameConfig,
```

Comment:
```suggestion
    val gameConfig: GameConfig,
```

`GameConfig` is immutable throughout the session and required to create an `InMemoryGame`